### ISSUE TITLE
Panic when imports use double quotes

### ISSUE THREAD

**Issue by codebreach** [Evertz/bzlgen#57]
For a source that looks like 
```ts
import { NgModule } from "@angular/core";
```

we get a stack trace:
```
 error     TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined 
    at assertPath (path.js:39:11)
    at Object.normalize (path.js:1120:5)
    at Workspace.isWorkspaceRelative (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/workspace.js:226:41)
    at Workspace.resolveRelativeToWorkspace (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/workspace.js:213:22)
    at NgGenerator.resolveLabelFromModuleSpecifier (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/generators/ts/ts.generator.js:100:60)
    at tsquery_1.tsquery.map (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/generators/ts/ts.generator.js:88:37)
    at Array.map (<anonymous>)
    at NgGenerator.processTsFileAst (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/generators/ts/ts.generator.js:88:18)
    at NgGenerator.processTsFile (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/generators/ng/ng.generator.js:82:18)
    at tsFiles.forEach (/home/madhav/.nvm/versions/node/v10.12.0/lib/node_modules/@evertz/bzlgen/src/generators/ng/ng.generator.js:58:53)
✖  error     Please report this error
```

root cause seems to be:

https://github.com/Evertz/bzlgen/blob/4a9ffd33fa0fb248cb56dad9501c0dbc65f96d40/src/generators/ts/ts.generator.ts#L104

Can be fixed by splitting on `"` and `'`

**Comment by mattem** [comment]
Thanks for the issue, opened #58 with a fix

**Comment by mattem** [comment]
Published as 0.5.1
